packages/twenty-server/src/database/typeorm/metadata/migrations/1721040829256-createFunctionMetadataTable.ts

```diff
@@ -0,0 +1,65 @@
+import { MigrationInterface, QueryRunner } from 'typeorm';
+
+export class CreateFunctionMetadataTable1721040829256
+  implements MigrationInterface
+{
+  name = 'CreateFunctionMetadataTable1721040829256';
+
+  public async up(queryRunner: QueryRunner): Promise<void> {
+    await queryRunner.query(
+      `CREATE TYPE "metadata"."functionMetadata_syncstatus_enum" AS ENUM('NOT_READY', 'READY')`,
+    );
+    await queryRunner.query(
+      `CREATE TABLE "metadata"."functionMetadata" ("id" uuid NOT NULL DEFAULT uuid_generate_v4(), "name" character varying NOT NULL, "syncStatus" "metadata"."functionMetadata_syncstatus_enum" NOT NULL DEFAULT 'NOT_READY', "workspaceId" uuid NOT NULL, "createdAt" TIMESTAMP WITH TIME ZONE NOT NULL DEFAULT now(), "updatedAt" TIMESTAMP WITH TIME ZONE NOT NULL DEFAULT now(), CONSTRAINT "IndexOnNameAndWorkspaceIdUnique" UNIQUE ("name", "workspaceId"), CONSTRAINT "PK_254107a6a35ecdf6f392dc63d72" PRIMARY KEY ("id"))`,
+    );
+    await queryRunner.query(
+      `ALTER TABLE "metadata"."workspaceMigration" ALTER COLUMN "name" SET NOT NULL`,
```

Comment:
do we want those?

Reply:
probably, think those are missing metadata updates

Reply:
will check